app/TextParser/TableCurrencySummary.php

```diff
@@ -70,7 +70,7 @@ public function process()
 								</tr>';
 				}
 				$html .= '<tr>
-								<td style="text-align:right;padding:0px 4px;">' . \App\Language::translate('LBL_AMOUNT', $this->textParser->moduleName) . '</td>
+								<td style="padding:0px 4px;font-weight:bold;">' . \App\Language::translate('LBL_AMOUNT', $this->textParser->moduleName) . '</td>
```

Comment:
don't use style

Reply:
I must, because YetiForcePDF doesn't support classes right now, it's the only way